ee/main.go

```diff
@@ -164,6 +164,7 @@ func main() {
 
 	modelDeployment := viper.GetBool("modelDeployment.enabled")
 	ingress := eecontrollers.PhIngress{}
+	primehubUrl := viper.GetString("primehubUrl")
```

Comment:
`primehubUrl` depends on `modelDeployment`

It would be better to move it under modelDeployment

```yaml
modelDeployment:
   enabled: true
   primehubUrl: http://foo-bar-dev-primehub.io
```